updating/index.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
```

Comment:
[Understanding {product-title} updates] does not match https://docs.openshift.com/container-platform/4.6/updating/understanding-the-update-service.html

Reply:
Addressed- Removed the wrong link. It was a residue from other version's PR

Reply:
ack